### ISSUE TITLE
Send Yoann ME70 Data from 2407RL

### ISSUE THREAD

**Issue by alicebeittel** [nmfs-swfsc-ast/ast-tasks#100]
@kstierhoff @jrenfree I got the 4TB hard drive from Yoann yesterday. I will work on transferring ME70 data from 2407RL either this Friday or early next week. They also asked for some meta data to go with it. I'll compile some of that too...that may take me a little longer. Thinking that could look like a list of transects with start/end times of the RAW files that have high catches of sardine, anchovy, mackerel, herring, etc nearby. Do we want to include the baby hake?

**Comment by kstierhoff** [comment]
The most time-critical and time-intensive part of this will be copying the files and shipping the drive. For metadata, what has he asked for? I offered to provide a file that had species-specific s_A per each 100-m interval, which would help him locate time periods where we encountered CPS. That's to say don't spend a lot of time in metadata if we have data in hand that will satisfy the metadata requested.

**Comment by jrenfree** [comment]
Agreed. Let's not waste time on the metadata, but focus on getting the raw ME70 data to him. Kevin could even just provide him plotSurvey or something which might give him most of the info he needs.

**Comment by alicebeittel** [comment]
We sent Yoann the data in Dec, I'll send him a check in to see how it's going and see what kind of metadata he is looking for.
